src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -311,6 +311,9 @@
   <data name="FancyZones_AppLastZoneMoveWindows.Content" xml:space="preserve">
     <value>Move newly created windows to their last known zone</value>
   </data>
+  <data name="FancyZones_OpenWindowOnActiveMonitor.Content" xml:space="preserve">
+    <value>Open newly created windows on active monitor</value>

```

Comment:
Let's use a similar wording as in `Move newly created windows to their last known zone` for consistency.

`Move newly created windows to the current active monitor`

We may have to add some extra information to clarify that it applies to windows that are not zoned as well, but for now it should be ok.

Reply:
Modified.